src/Microsoft.ML.GenAI.Phi/README.md

```diff
@@ -0,0 +1,119 @@
+# Microsoft.ML.GenAI.Phi
+Torchsharp implementation of Microsoft phi-series models for GenAI
+
+## Supported list
```

Comment:
For the ones that aren't checked, do we have a path to support them? If not, maybe remove them

Reply:
Yes, I am working on implementing those models